### DISCUSSION TITLE
Can I hide the toolbar? I just need to show the canvas

### DISCUSSION THREAD

**Discussion by NILISSSSSSSSSSSSSSSSS** [epam/ketcher#861] [General]
![image](https://user-images.githubusercontent.com/29522483/138389235-fb85e885-af59-4b29-9a8a-820c1610a4e2.png)

**Comment by AndreiMazol** [comment]
Hi @NILISSSSSSSSSSSSSSSSS ,

Left toolbar items and server buttons can be hidden by configuring <Editor> component. Please look at https://github.com/epam/ketcher/discussions/710 for details.

Best regards,
Andrei

**Comment by AndreiMazol** [comment]
Dear @NILISSSSSSSSSSSSSSSSS ,

unfortunately hiding of all  elements is not supported right now. We plan to rework components library that is delivered as a part of ketcher-react package so in the future releases it will be possible to create your custom editor.
